core/api/test/integration/services/openai.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { Assistant } from "@/services/openai"
+describe("ChatSupport", () => {
+  it("add 2 message to the thread", async () => {
+    // this test needs an OpenAI API key to run
+    // not running this test in CI
+    if (!process.env.OPENAI_API_KEY) {
+      console.log("No OpenAI API key found, skipping test")
+      return
+    }
```

Comment:
this is a bad practice.. if the test is necessary please create the mock, other wise I prefer this test to be removed

Reply:
see that more as a local dev tool?

Reply:
no, I mean, this is logic that avoid the test.. tests must test something